From b78583a339e2064df19986f1a702fe21e1e9e215 Mon Sep 17 00:00:00 2001
From: Oleg Karfich <oleg.karfich@wago.com>
Date: Fri, 17 Nov 2017 11:38:07 +0100
Subject: [PATCH] dts: pfcxxx: change rt-prio of watchdog thread

	push the watchdog thread to rt prio that is
	above bas-keys
---
 arch/arm/boot/dts/wago-pfc100.dtsi   | 2 +-
 arch/arm/boot/dts/wago-pfc200.dtsi   | 2 +-
 arch/arm/boot/dts/wago-pfc200v2.dtsi | 2 +-
 arch/arm/boot/dts/wago-pfc200v3.dtsi | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/wago-pfc100.dtsi b/arch/arm/boot/dts/wago-pfc100.dtsi
index 114a5604..a071aa7 100644
--- a/arch/arm/boot/dts/wago-pfc100.dtsi
+++ b/arch/arm/boot/dts/wago-pfc100.dtsi
@@ -146,7 +146,7 @@
 		hw_margin_ms = <1600>;
 		status = "okay";
 
-		rt-prio = <99>;
+		rt-prio = <91>;
 	};
 };
 
diff --git a/arch/arm/boot/dts/wago-pfc200.dtsi b/arch/arm/boot/dts/wago-pfc200.dtsi
index 4acc7eb..e91b92b 100644
--- a/arch/arm/boot/dts/wago-pfc200.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200.dtsi
@@ -275,7 +275,7 @@
 		hw_margin_ms = <1600>;
 		status = "okay";
 
-		rt-prio = <99>;
+		rt-prio = <91>;
 	};
 };
 
diff --git a/arch/arm/boot/dts/wago-pfc200v2.dtsi b/arch/arm/boot/dts/wago-pfc200v2.dtsi
index 06afca0..43f0180 100644
--- a/arch/arm/boot/dts/wago-pfc200v2.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200v2.dtsi
@@ -132,7 +132,7 @@
 		hw_margin_ms = <1600>;
 		status = "okay";
 
-		rt-prio = <99>;
+		rt-prio = <91>;
 	};
 };
 
diff --git a/arch/arm/boot/dts/wago-pfc200v3.dtsi b/arch/arm/boot/dts/wago-pfc200v3.dtsi
index 5d4cee5..ad9ba9c 100644
--- a/arch/arm/boot/dts/wago-pfc200v3.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200v3.dtsi
@@ -128,7 +128,7 @@
 		hw_margin_ms = <1600>;
 		status = "okay";
 
-		rt-prio = <99>;
+		rt-prio = <91>;
 	};
 };
 
-- 
2.7.4

